package.json

```diff
@@ -35,12 +35,24 @@
     "@emotion/react": "^11.14.0",
     "@emotion/styled": "^11.14.0",
     "@fontsource/roboto": "^5.1.1",
+    "@lexical/react": "^0.23.1",
+    "@lexical/rich-text": "^0.23.1",
     "@mui/icons-material": "^6.4.0",
     "@mui/lab": "^6.0.0-beta.23",
     "@mui/material": "^6.4.0",
     "@mui/x-date-pickers": "^7.23.6",
     "@reduxjs/toolkit": "^2.5.0",
     "@tanstack/react-query": "^5.64.1",
+    "@tiptap/extension-bold": "^2.11.2",
+    "@tiptap/extension-document": "^2.11.2",
+    "@tiptap/extension-history": "^2.11.3",
+    "@tiptap/extension-italic": "^2.11.3",
+    "@tiptap/extension-link": "^2.11.3",
+    "@tiptap/extension-paragraph": "^2.11.2",
+    "@tiptap/extension-placeholder": "^2.11.3",
+    "@tiptap/extension-text": "^2.11.2",
+    "@tiptap/extension-underline": "^2.11.3",
+    "@tiptap/react": "^2.11.2",
```

Comment:
usj, kommer til å bli et styr når dependabot kommer med 10 PRs for hver nye versjon.
Har det samme med Uppy, hvor vi kan få 8 PRs. Tror det er mulig å konfigurere dependabot sånn at alle samles i en PR da, så det er kanskje noe vi burde vurdere for de pakkene det gjelder